server/routerlicious/packages/services-utils/src/morganLoggerMiddleware.ts

```diff
@@ -106,13 +126,23 @@ export function jsonMorganLoggerMiddleware(
 			if (computeAdditionalProperties) {
 				additionalProperties = computeAdditionalProperties(tokens, req, res);
 			}
+			let statusCode = tokens.status(req, res);
+			if (!statusCode) {
+				if (res.locals.serverTimeout) {
+					statusCode = "Server Timeout";
+				} else if (res.locals.clientDisconnected) {
+					statusCode = "499";
+				} else {
+					statusCode = "STATUS_UNAVAILABLE";
```

Comment:
Nit: I wonder if we should change this to 504 too?

Reply:
I can but I prefer to leave `STATUS_UNAVAILABLE` for following reasons:
1. `STATUS_UNAVAILABLE` is a known status code already in our logging system and perhaps some monitors are based on that.
2. Using `STATUS_UNAVAILABLE` for these unknown results would give us the chance to further drill down what are the detailed reason for this. Using 504 may hide it with real 504.

Therefore, I think it may be better to leave `STATUS_UNAVAILABLE` for now.